src/styles.css

```
@@ -1968,6 +1968,10 @@ button.rematch-decline::before {
     font-size: 1.5em !important;
 }
 
+.storm--end__stats th {
+    padding: 10px;
+}
+
```

```suggestion
.storm--end__stats th {
    padding-left: 10px !important;
}

```
I would prefer `padding-left` here because it is really only about the padding to the left and not the global padding. Also, as always, we should use `!important` to make sure that the rule is really applied.